### PR TITLE
Fix issue #6212 POD Generated podfile doesn't include all generated swift files #6212

### PR DESCRIPTION
**Pull request by hce**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [X] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [X] Filed the PR against the correct branch: master for non-breaking changes and `3.0.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

Suggested fix for issue described in issue #6212

### CODE REVIEW

**Comment by hce** [comment]
Looks like the travis error isn't related to the patch.

"[ERR!] GitHub API: GitHub rate limit reached."

**Comment by wing328** [comment]
@hce thanks for the PR. Please ignore the Github error as the latest master should have addressed it.

cc @jgavris @Edubits @jaz-ah

File: samples/client/petstore/swift4/default/.swagger-codegen/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.2.3-SNAPSHOT
\ No newline at end of file
+2.3.0-SNAPSHOT
```
Comment (jgavris) [inline samples/client/petstore/swift4/default/.swagger-codegen/VERSION]: This PR should be against `2.3.0` branch yes?

Comment (wing328) [inline samples/client/petstore/swift4/default/.swagger-codegen/VERSION]: Master is now 2.3.0 (as we released 2.2.3 about 2 weeks ago)

Comment (jgavris) [inline samples/client/petstore/swift4/default/.swagger-codegen/VERSION]: I'm a little behind then, great news!

**Comment by wing328** [comment]
Tests passed via https://travis-ci.org/swagger-api/swagger-codegen/builds/260671388

**Pull request merged**